Changelog.md

```diff
@@ -3,12 +3,34 @@
 * [New sized and/or unsigned variants of `readInt` and `readInteger`](https://github.com/haskell/bytestring/pull/438)
 * [`readInt` returns `Nothing`, if the sequence of digits cannot be represented by an `Int`, instead of overflowing silently](https://github.com/haskell/bytestring/pull/309)
 * [Remove `zipWith` rewrite rule](https://github.com/haskell/bytestring/pull/387)
-* [Export `unsafeIndex` for ShortByteString which had been accidentally removed in v0.11.3.0](https://github.com/haskell/bytestring/pull/532)
 * [`ShortByteString` is now a wrapper over boxed `Data.Array.Byte.ByteArray` instead of unboxed `ByteArray#` directly](https://github.com/haskell/bytestring/pull/410)
 * [`fromListN` from `instance IsList ShortByteString` throws an exception if the first argument does not match the length of the second instead of silent ignore](https://github.com/haskell/bytestring/pull/410)
 
+[0.12.0.0]: https://github.com/haskell/bytestring/compare/0.11.4.0...0.12.0.0
+
+[0.11.4.0] — January 2023
 
-[0.12.0.0]: https://github.com/haskell/bytestring/compare/0.11.3.0...0.12.0.0
+* [Export `unsafeIndex` for ShortByteString which had been accidentally removed in v0.11.3.0](https://github.com/haskell/bytestring/pull/532)
+* [Simplify `useAsCString`](https://github.com/haskell/bytestring/pull/516)
+* [Remove redundant comparisons in `Data.ByteString.Short.splitAt`](https://github.com/haskell/bytestring/pull/528)
+* [Speed up `Builder`s for non-host endianness](https://github.com/haskell/bytestring/pull/531)
+* [Document possible interleaving of `hPutStrLn` and friends](https://github.com/haskell/bytestring/pull/518)
+* [Documentation tweaks](https://github.com/haskell/bytestring/pull/523)
+* [Add lower bound for `tasty-quickcheck`](https://github.com/haskell/bytestring/pull/520)
+* [Improve behavior of `Builder` literals at chunk boundaries](https://github.com/haskell/bytestring/pull/538)
```

Comment:
More like a bugfix than an improvement, I think. Let's advertise it higher, just after `unsafeIndex` item, it's a pretty big deal which caused trouble for users.

Reply:
The old behavior was arguably "correct," but I agree that this is de-facto a bug-fix and have re-phrased the entry accordingly. I also split the entries into sections while I was moving this one.